org.eclipse.jgit/src/org/eclipse/jgit/util/StringUtils.java

```diff
@@ -334,4 +335,20 @@ public static String replaceLineBreaksWithSpace(String in) {
 		}
 		return new String(buf, 0, o);
 	}
+
+	/**
+	 * Replace LF with LF and single space.
+	 *
+	 * @param text
+	 *            A string with line breaks
+	 * @return in with line breaks and spaces in new lines
+	 * @since 5.2
+	 */
+	public static String replaceLFWithLFSpace(final String text) {
```

Comment:
This is a red flag. You shouldn't needs this at all for formatting Git objects. There's gotta be a better way in JGit.

Reply:
In that case will just pull it into PGPSign class as BC does not provide signature which aligns with https://github.com/git/git/blob/53f9a3e157dbbc901a02ac2c73346d375e24978c/Documentation/technical/signature-format.txt#L79,L89